package.json

```diff
@@ -114,6 +114,7 @@
     "@metamask/obs-store": "^5.0.0",
     "@metamask/post-message-stream": "^4.0.0",
     "@metamask/providers": "^8.1.1",
+    "@metamask/smart-transactions-controller": "file:./../smart-transactions-controller/metamask-smart-transactions-controller-v0.0.1.tgz",
```

Comment:
We need to publish latest changes of the STX Controller and then we can use the `@metamask/smart-transactions-controller` node module instead of a .tgz file.